msg/servo_status.msg

```diff
@@ -0,0 +1,28 @@
+uint64 timestamp					# time since system start (microseconds)
+uint8 CONNECTED_SERVO_MAX = 8				# The number of ESCs supported. Current (Q2/2013) we support 8 ESCs
+
+uint8 SERVO_CONNECTION_TYPE_PPM = 0			# Traditional PPM ESC
```

Comment:
I would drop these until we actually need them, many of these connection types will never be real.

Reply:
Cool, I have dropped these.